packages/lesswrong/server/editor/make_editable_callbacks.ts

```diff
@@ -221,7 +221,7 @@ export async function ckEditorMarkupToHtml(markup: string): Promise<string> {
 async function dataToHTML(data, type, sanitizeData = false) {
   switch (type) {
     case "html":
-      return sanitizeData ? sanitize(data) : data
+      return sanitizeData ? sanitize(data) : await mjPagePromise(data, trimLatexAndAddCSS)
```

Comment:
(I verified that all calls to this are inside write operations, so pageload performance will not be affected.)